sunpy/io/tests/test_fits.py

```diff
@@ -176,3 +177,11 @@ def test_warn_longkey():
                                'goodkey': 'test'})
     assert 'GOODKEY' in fits.keys()
     assert 'BADLONGKEY' not in fits.keys()
+
+
+def test_read_memmap():
+    # Check that memmap is read correctly from a file
+    data, header = sunpy.io._fits.read(TEST_AIA_IMAGE, memmap=True)[0]
```

Comment:
As the default for astropy's fits reader is to use memmap, this test would also pass if `memmap=True` was not included in this function call. Should we also be testing that `memmap=False` *doesn't* do memory mapping?

Reply:
Yes, we definitely should test `memmap=False` as well.

Reply:
Sure.